pysph/base/tests/test_opencl.py

```diff
@@ -4,8 +4,47 @@
 
 pytest.importorskip("pysph.base.opencl")
 
+import pyopencl as cl
+
 from pysph.base.utils import get_particle_array  # noqa: E402
-from pysph.base.opencl import DeviceHelper  # noqa: E402
+from pysph.base.opencl import DeviceHelper, DeviceArray  # noqa: E402
+
+
+class TestDeviceArray(TestCase):
+    def setUp(self):
+        self.ctx = cl.create_some_context(interactive=False)
```

Comment:
Again, no need for this, if you see the other tests I don't bother with the queue, see the `TestDeviceHelper` class.